src/libsync/configfile.h

```diff
@@ -117,6 +117,12 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     void setNewBigFolderSizeLimit(bool isChecked, quint64 mbytes);
     bool confirmExternalStorage() const;
     void setConfirmExternalStorage(bool);
+    /** delta sync */
+    bool deltaSyncEnabled() const;
+    void setDeltaSyncEnabled(bool enabled);
+    quint64 deltaSyncMinFileSize() const;
```

Comment:
Actually no, because `0` does the opposite, if enabled that means it will be allowed for **all** file sizes. You wouldn't be able to do that if `0` means disabled.